charts/consul/Chart.yaml

```diff
@@ -22,7 +22,7 @@ annotations:
     - name: consul-dataplane
       image: docker.mirror.hashicorp.services/hashicorppreview/consul-dataplane:1.3.0
     - name: envoy
-      image: envoyproxy/envoy:v1.26.2
+      image: envoyproxy/envoy:v1.25.11
```

Comment:
x-post for posterity: https://github.com/hashicorp/consul-k8s/pull/3116#discussion_r1368872800